utils/log_writer.go

```diff
@@ -0,0 +1,22 @@
+package utils
+
+import (
+	"bufio"
+	"io"
+
+	log "github.com/Sirupsen/logrus"
+)
+
+// NewLogWriter pipes stdout/err logs to logrus
+func NewLogWriter(fields log.Fields, level log.Level) io.WriteCloser {
```

Comment:
Ok, those levels make sense then... I think what I might be getting distracted by is that whatever's coming out of stdout/stderr is either unstructured or structured by the application. If the application is sending unstructured logs then this feature lets us provide structure to them, which is nice. But if the application is sending structured logs then we have opportunities to drop logs unintentionally.

Suppose our application is doing structured logging at the DEBUG level but ContainerPilot is configured for INFO. When the logs get shipped off to a log collector, all the DEBUG logs are going to be logged at INFO in the collector. Maybe it's just a matter of telling end-users they shouldn't use the `text` option if the application sends structured logs? Kinda sucks that we then drop the structure from ContainerPilot logs in that scenario though.